gym/wrappers/test_flatten_observation.py

```diff
@@ -19,12 +19,12 @@ def test_flatten_observation(env_id):
         space = spaces.Tuple(
             (spaces.Discrete(32), spaces.Discrete(11), spaces.Discrete(2))
         )
-        wrapped_space = spaces.Box(-np.inf, np.inf, [32 + 11 + 2], dtype=np.float32)
+        wrapped_space = spaces.Box(-np.inf, np.inf, [32 + 11 + 2], dtype=np.float64)
```

Comment:
```suggestion
        wrapped_space = spaces.Box(0, 1, [32 + 11 + 2], dtype=np.int64)
```

This doesn't have too much impact on the test, but this should be `dtype=np.int64` (the flattened space takes the dtype of `Discrete`, [which is `np.int64`](https://github.com/openai/gym/blob/150404104afc57ea618acca34d2d1f797d779869/gym/spaces/discrete.py#L17)). Also the bounds should be 0 and 1 ([according to the rules for flattening the space](https://github.com/openai/gym/blob/150404104afc57ea618acca34d2d1f797d779869/gym/spaces/utils.py#L200-L214), since flattening a `Tuple` of `Dsicrete` spaces leads to multi-hot vectors).